docs/frontend.py

```diff
@@ -84,8 +84,8 @@ def docker_run(**kwargs):
     if os.isatty(os.isatty(sys.stdout.fileno())):
         cmd.append("-it")
     cmd.extend(["wpt:docs", "./wpt"])
-    if kwargs["venv"]:
-        cmd.extend(["--venv", kwargs["venv"]])
+    # /app/venv is created and activated in the Dockerfile
```

Comment:
FWIW I think talking about something being "activated in the Dockerfile" is kind of confusing. Maybe this comment could read `# /app/venv is created during docker build and is automatically active inside the container`

Reply:
I don't think "activated in the Dockerfile" is inaccurate, but let's go with your wording.